app/src/renderer/components/governance/LiProposal.vue

```diff
@@ -31,16 +31,16 @@
     </td>
     <td>{{ `#` + proposal.proposal_id }}</td>
     <td class="li-proposal__value yes">
-      {{ tally.yes }}
+      {{ tally.yes || `--` }}
```

Comment:
side note: we should maybe prettify the tally result numbers

Reply:
what do you mean by prettify?

Reply:
there is a function called `pretty` which shortens the value after 4 decimals